src/turret.cpp

```diff
@@ -78,6 +78,21 @@ turret_data vehicle::turret_query( const tripoint &pos ) const
     return const_cast<vehicle *>( this )->turret_query( pos );
 }
 
+bool vehicle::is_manual_turret( vehicle_part &pt )
+{
+    for( const int elem : parts_at_relative( pt.mount, false ) ) {
+        if( part_info( elem ).has_flag( "MANUAL" ) ) {
+            return true;
+        }
+    }
+    return false;
+}
```

Comment:
```suggestion
bool vehicle::is_manual_turret(const vehicle_part &pt)
{
    const auto &parts = parts_at_relative(pt.mount, false);
    return std::any_of(parts.begin(), parts.end(), 
        [this](int elem) { return part_info(elem).has_flag("MANUAL"); });
}
```
let's use [std::any_of](https://en.cppreference.com/w/cpp/algorithm/all_any_none_of)